common/variables.nix

```diff
@@ -4,13 +4,36 @@ rec {
   webtreeDir = "/storage/webtree";
   homesDir = "/storage/home";
 
+  # This maps some domains to other domains
+  # usually due to broken DNS entries
+  # Allows certs to be correctly generated
+  brokenDomains = {
+    "djbdns.now.ie" = "djbdns.now.ie";
+    "romana.now.ie" = "djbdns.now.ie";
+    "www.luxgaa.lu" = "www.luxgaa.lu";
+    "www.iahpc.ie" = "www.iahpc.ie";
+    "techweek.dcu.ie" = "techweek.dcu.ie";
```

Comment:
mind explaining this a bit?

Reply:
Part of the script for generating certs will break down domains to their tld. So for example, www.iahpc.ie will become iahpc.ie. In most cases, this is totally fine, but for these few sites their tld's do not point at redbrick and thus we can't generate HTTP-validated certs for them. This list allows us to work around this issue. I might add a more descriptive comment.